Engine/DataFeeds/SubscriptionSynchronizer.cs

```diff
@@ -145,6 +143,11 @@ public TimeSlice Sync(DateTime frontier, IEnumerable<Subscription> subscriptions
                         }
                     }
 
+                    if (packet.Count > 0 && !packet.Configuration.Symbol.IsUniverseSymbol())
```

Comment:
The subscription defines whether or not it's a universe subscription [here](https://github.com/QuantConnect/Lean/blob/master/Engine/DataFeeds/Subscription.cs#L79).

For posterity: This change will prevent universe subscription data from making its way into the `Slice` object.